packages/vitest/src/node/config/resolveConfig.ts

```diff
@@ -56,7 +56,7 @@ function parseInspector(inspect: string | undefined | boolean | number) {
   return { host, port: Number(port) || defaultInspectPort }
 }
 
-export function resolveApiServerConfig<Options extends ApiConfig & UserConfig>(
+export function resolveApiServerConfig<Options extends ApiConfig & Omit<UserConfig, 'expect'>>(
```

Comment:
This is provided with the `browser` object, the new `browser.expect` property conflicts with `expect`. I noticed `options.expect` is not used tho, so instead of removing it at runtime I removed it from the type.